indra/llui/llemojihelper.cpp

```diff
@@ -0,0 +1,166 @@
+/**
+* @file llemojihelper.h
```

Comment:
Every so minor but the @file comment here should match the actual filename.

Reply:
Fixed.